16:45:29 [INF][   WBContext UI]: Initializing workbench context UI with these values:
	base dir: /usr/share/mysql-workbench
	plugin path: /usr/lib/mysql-workbench/plugins
	struct path: /usr/share/mysql-workbench/grt
	module path: /usr/lib/mysql-workbench/modules
	library path: /usr/share/mysql-workbench/libraries
	user data dir: /home/yasmim/.mysql/workbench
	open at start: 
	open type: 
	run at startup: 
	run type: 
	Force SW rendering: No
	Force OpenGL: No
	quit when done: No
16:45:29 [INF][      WBContext]: WbContext::init
16:45:31 [INF][            WBA]: Looking for extension modules for WBA...
16:45:31 [INF][            WBA]: 0 extension modules found
16:45:32 [WRN][            grt]: /home/yasmim/.mysql/workbench/connections.xml:25: link '5aa61a90-ae89-11e7-aed1-e0db55ff19d2' <object GrtObject> key=owner could not be resolved
16:45:33 [INF][      WBContext]: System info:
 	MySQL Workbench Community (GPL) for Linux/Unix version 6.3.9 CE build 10690321 (64 bit)
	Configuration Directory: /home/yasmim/.mysql/workbench
	Data Directory: /usr/share/mysql-workbench
	Cairo Version: 1.14.6
	OS: Ubuntu 16.04 x86_64
	CPU: 4x Intel(R) Core(TM) i7-4510U CPU @ 2.00GHz (2383.386MHz) - 7,70GiB RAM
No video adapter info available
	Distribution: Ubuntu 16.04.3 LTS

	Fips mode enabled: no

16:45:33 [INF][    WQE backend]: Found auto-save workspace /home/yasmim/.mysql/workbench/sql_workspaces/web-2.autosave
17:09:49 [ERR][SQL Editor Form]: SqlEditorForm: exception in do_connect method: Exception: Lost connection to MySQL server at 'reading initial communication packet', system error: 0
17:09:49 [WRN][SQL Editor Form]: Lost connection to MySQL server at 'reading initial communication packet', system error: 0
17:09:49 [INF][SQL Editor Form]: Error 2013 connecting to server, assuming server is down and opening editor with no connection
17:09:49 [INF][SQL Editor Form]: Error 2013 connecting to server, assuming server is down and opening editor with no connection
17:10:09 [ERR][wb_admin_control.py:init:314]: Error connecting to MySQL: Lost connection to MySQL server at 'reading initial communication packet', system error: 0 (code 2013)
17:10:10 [INF][wb_admin_control.py:query_server_installation_info:868]: Currently connected to MySQL server version 'unknown', conn status = None, active plugins = []
17:10:10 [INF][wb_admin_control.py:acquire_admin_access:288]: Target OS detection returned: os_type=linux, os_name=Linux, os_variant=Ubuntu, os_version=16.04
17:10:10 [INF][wb_admin_control.py:exec_query:631]: sql connection is down
17:10:30 [ERR][wb_admin_control.py:server_polling_thread:564]: Error creating SQL connection for monitoring: MySQLError("Lost connection to MySQL server at 'reading initial communication packet', system error: 0 (code 2013)",)
17:10:43 [ERR][wb_admin_main.py:shutdown:219]: shutting down admn
17:12:18 [ERR][wb_admin_control.py:init:314]: Error connecting to MySQL: Lost connection to MySQL server at 'reading initial communication packet', system error: 0 (code 2013)
17:12:18 [INF][wb_admin_control.py:exec_query:631]: sql connection is down
17:12:31 [ERR][wb_admin_main.py:shutdown:219]: shutting down admn
17:12:31 [ERR][               ]: atk_selection_ref_selection: assertion 'ATK_IS_SELECTION (obj)' failed
17:12:38 [ERR][wb_admin_control.py:server_polling_thread:564]: Error creating SQL connection for monitoring: MySQLError("Lost connection to MySQL server at 'reading initial communication packet', system error: 0 (code 2013)",)
17:13:05 [ERR][wb_admin_control.py:init:314]: Error connecting to MySQL: Lost connection to MySQL server at 'reading initial communication packet', system error: 0 (code 2013)
17:13:05 [ERR][               ]: atk_selection_ref_selection: assertion 'ATK_IS_SELECTION (obj)' failed
17:13:05 [INF][wb_admin_control.py:query_server_installation_info:868]: Currently connected to MySQL server version 'unknown', conn status = None, active plugins = []
17:13:05 [INF][wb_admin_control.py:acquire_admin_access:288]: Target OS detection returned: os_type=linux, os_name=Linux, os_variant=Ubuntu, os_version=16.04
17:13:05 [INF][wb_admin_control.py:exec_query:631]: sql connection is down
17:13:25 [ERR][wb_admin_control.py:server_polling_thread:564]: Error creating SQL connection for monitoring: MySQLError("Lost connection to MySQL server at 'reading initial communication packet', system error: 0 (code 2013)",)
17:13:30 [ERR][wb_admin_main.py:shutdown:219]: shutting down admn
17:13:47 [WRN][            grt]: /home/yasmim/.mysql/workbench/bd.mwbd/document.mwb.xml:64: link '48b18b10-b693-11e7-b49c-4851b78d005e' <object db.Schema> key=defaultSchema could not be resolved
17:14:18 [WRN][            grt]: /home/yasmim/.mysql/workbench/bd.mwbd/document.mwb.xml:64: link '48b18b10-b693-11e7-b49c-4851b78d005e' <object db.Schema> key=defaultSchema could not be resolved
17:15:08 [INF][  column_widths]: Initializing column width cache for unconnected
17:15:20 [WRN][   WBContext UI]: Server instance  has no attached connection, deleting it
17:15:20 [WRN][   WBContext UI]: Server instance  has no attached connection, deleting it
17:15:20 [WRN][   WBContext UI]: Server instance  has no attached connection, deleting it
17:18:34 [INF][   WBContext UI]: Opening Migration Wizard...
17:18:35 [INF][      WBContext]: Loaded 0/0 new non-MySQL connections
18:12:36 [INF][   WBContext UI]: Opening Migration Wizard...
18:12:52 [INF][         Wizard]: Execute 'Connect to source DBMS'
18:12:52 [INF][            grt]: Connecting to Mysql@localhost:3306...	
18:12:55 [INF][            grt]: Connecting to Mysql@localhost:3306...	
18:12:55 [INF][            grt]: Connected to Mysql@localhost:3306, 5.7.19-0ubuntu0.16.04.1	
18:12:55 [INF][         Wizard]: Execute 'Check target DBMS connection'
18:12:55 [INF][            grt]: Connecting to Mysql@localhost:3306...	
18:12:55 [INF][            grt]: Connected	
18:12:55 [INF][         Wizard]: Starting thread for 'Retrieve schema list from source'
18:13:04 [INF][         Wizard]: Execute 'Connect to source DBMS'
18:13:04 [INF][            grt]: Connecting to Mysql@localhost:3306...	
18:13:04 [INF][            grt]: Connected to Mysql@localhost:3306, 5.7.19-0ubuntu0.16.04.1	
18:13:04 [INF][         Wizard]: Starting thread for 'Reverse engineer selected schemas'
18:13:04 [INF][            grt]: Reverse engineering mydb from def	
18:13:05 [INF][            grt]: Reverse engineering tables from mydb	
18:13:05 [INF][            grt]: Reverse engineering triggers from mydb	
18:13:05 [INF][            grt]: Reverse engineering stored procedures from mydb	
18:13:05 [INF][            grt]: Reverse engineering functions from mydb	
18:13:05 [INF][         Wizard]: Execute 'Post-processing of reverse engineered schemas'
18:13:07 [INF][         Wizard]: Starting thread for 'Migrate Selected Objects'
18:13:08 [INF][         Wizard]: Starting thread for 'Generate SQL CREATE Statements'
18:13:08 [WRN][   mforms.linux]: Unknown column storage typeUnknown column storage typeUnknown column storage typeOpened connection 'Local instance 3306' to (Ubuntu) version 5.7.19-0ubuntu0.16.04.1
18:14:57 [ERR][ sqlide-history]: Can't open SQL history file /home/yasmim/.mysql/workbench/sql_history/2017-10-21
18:16:02 [ERR][SQL Editor Form]: There are pending changes. Please commit or rollback first.
18:16:20 [ERR][SQL Editor Form]: 1452: Cannot add or update a child row: a foreign key constraint fails (`mydb`.`Agricultor`, CONSTRAINT `fk_Agricultor_PessoaFisica` FOREIGN KEY (`PessoaFisica_CPF`) REFERENCES `PessoaFisica` (`CPF`) ON DELETE NO ACTION ON UPDATE NO ACTION)
18:16:20 [ERR][  GRTDispatcher]: exception in grt execute_task, continuing: Exception: There was an error while applying the SQL script to the database.
18:16:20 [ERR][  GRTDispatcher]: worker: task 'wizard task' has failed with error:.There was an error while applying the SQL script to the database.
